cmd/orb.go

```diff
@@ -255,7 +255,8 @@ func validateOrb(opts orbOptions) error {
 		return err
 	}
 
-	opts.log.Infof("Orb at `%s` is valid.", opts.args[0])
+	opts.log.Infof("Orb input is valid.")
```

Comment:
Seems a bit of a shame to actually lose the file info, especially if using this inside a build. Not a reason to hold this up, but pointing out that there was some usefulness to seeing the path in the output. If we kept it I guess we'd have to special-case "-".

Reply:
Yeah, I would like top see `orb at <path> is valid` or `orb input is valid` here, depending on whether input is from a file or stdin.